docs/build_docs.sh

```diff
@@ -72,8 +74,34 @@ while getopts "piez" opt; do
 		z)
 		JEKYLL_CONFIG="--config _config.yml,_config_dev_zh.yml"
 		;;
+		*) echo "usage: $0 [-e|-z] [-i|-p]" >&2
+		exit 1 ;;
 	esac
 done
 
 # use 'bundle exec' to insert the local Ruby dependencies
-bundle exec jekyll ${JEKYLL_CMD} ${JEKYLL_CONFIG} --source "${DOCS_SRC}" --destination "${DOCS_DST}"
+
+if [ "${JEKYLL_CMD}" = "build" ] && [ -z "${JEKYLL_CONFIG}" ]; then
+  # run parallel builds for all languages if not serving or creating a single language only
+
+  # run processes and store pids
+  echo "Spawning parallel builds for ${DOC_LANGUAGES}..."
+  pids=
```

Comment:
my god is this actually valid in bash? It looks soo much like an incomplete statement.

Reply:
yes it it ;)
but I can also replace it with `pids=""`